src/mavric/src/900MHz_Interface.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+import rospy
+import time
+import os
+from std_msgs.msg import Int32, String
+from pySerialTransfer import pySerialTransfer as txfer
+import driveMath
+
+
+from mavric.msg import Drivetrain
+from mavric.msg import Steertrain
+
+port = '/dev/ttyACM0'
+
+# Defaults and beginning values
+drive = 0.0
+steer = 0.0
+mode = 0
+parameters = [0,0,0,0,0,0,0,0,0,0]
+deadzone = 0.05
+sensitivity = 0.5
+LoRaEnabled = False
+
+class struct(object):
+    drive = float
+    steer = float
+    mode = float
+
+
+# Client Callback function that will run whenever the value for clients connected changes. 
+# if connected clients is greater than zero, publish all zeros once and stop publishing to 
+# stop interrupting websocket clients.
+def client_cb(data):
+    global LoRaEnabled
+    val = int(data.data)
+    if val == 0:
+        LoRaEnabled = True
+    else:
+        if LoRaEnabled == True:
+            drive.publish(0.0,0.0,0.0,0.0,0.0,0.0)
+            steer.publish(0.0,0.0,0.0,0.0)
+        LoRaEnabled = False
+
+
+if __name__=='__main__':
+    try:
+        # Publishers and Subscribers
+        drive = rospy.Publisher("Drive_Train", Drivetrain, queue_size=10)
+        steer = rospy.Publisher("Steer_Train", Steertrain, queue_size=10)
+        status = rospy.Publisher("LoRa_Status", String, queue_size=10)
+        clients = rospy.Subscriber("client_count", Int32, client_cb, queue_size=10)
+        rospy.init_node("LoRa_Radio")
+
+        dataRX = struct
+        link = txfer.SerialTransfer(port)   # setup serial transfer on port specified
+        link.open() # open link between rover and LoRa
+        time.sleep(1)
```

Comment:
is there a need to sleep here?

Reply:
A sleep is used to make sure the serial transfer library opens the link fully and ensures the handshake is okay

Reply:
The 1 second was randomly chosen, I bet we can get that down